Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -482,8 +490,8 @@ class EmbeddedUITests: PaymentSheetUITestCase {
         // Try to fill a card
         app.buttons["New card"].waitForExistenceAndTap()
         XCTAssertTrue(app.staticTexts["Add new card"].waitForExistence(timeout: 10))
-        try! fillCardData(app, cardNumber: "5555555555554444", postalEnabled: true)
-        app.toolbars.buttons["Done"].waitForExistenceAndTap()
+        XCTAssertTrue(app.buttons["New card"].isSelected) // Should show selected, the form should be filled with a valid card from above
```

Comment:
This should now show as selected since we are presenting the form with a filled out card form (valid payment option)